app/api/v1/models/ballot.py

```diff
@@ -17,14 +32,28 @@
 ]
 
 DecryptionShare = Any
-SubmittedBallot = Any
-CiphertextBallot = Any
+AnySubmittedBallot = Any
+AnyCiphertextBallot = Any
```

Comment:
I've had a heck of a time differentiating classes and aliases that have identical names.  Hopefully these aliases will eventually all disappear and be replaced with DTO's, but in the meantime I will at least prefix ones that I touch with the word Any to avoid collisions.

Reply:
I feel ya on this.